multi_import/multi_file_importer.py

```diff
@@ -82,11 +84,23 @@ def pre_read(self, file_object):
         return file_object
 
 
+class TxtFormat(FileFormat):
+    def __init__(self):
+        super(TxtFormat, self).__init__(txt, read_file_as_string=True)
+
+    def write(self, dataset):
```

Comment:
I just checked the `_registerformats` method of `DataSet` [here](https://github.com/kennethreitz/tablib/blob/master/tablib/core.py#L252).

We should replace change the `write` method of `FileFormat` to the following:

```
def write(self, dataset):
    data = self.format.export_set(dataset)
    f = BytesIO()
    f.write(data)
    return f
```

Then we don't need `TxtFormat` class.